package.json

```diff
@@ -11,14 +11,18 @@
     "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
     "bootstrap": "^4.5.0",
     "jquery": "^3.5.1",
```

Comment:
Try not to use pacakges like these. especially Jquery. I see that you did not use Jquery. So try not to include this as package.